protoc-gen-grpc-gateway/gengateway/template.go

```diff
@@ -219,6 +220,14 @@ var _ io.Reader
 var _ status.Status
 var _ = runtime.String
 var _ = utilities.NewDoubleArray
+
+func addDataToContext(ctx context.Context, method, urlTemplate string, start time.Time) context.Context {
+	ctx = context.WithValue(ctx, "executionStartedAt", start)
+	ctx = context.WithValue(ctx, "requestMethod", method)
+	ctx = context.WithValue(ctx, "urlPathTemplate", urlTemplate)
```

Comment:
all this lines can be merged into one. define struct and use it's type like a key